CleverTap/Runtime/Native/UnityNativeWrapper/Validators/UnityNativeEventValidator.cs

```diff
@@ -166,6 +168,46 @@ private bool IsAnyNumericType(object o) {
             return o is short || o is int || o is long ||
                    o is float || o is double || o is decimal;
         }
+
+        /**
+            * Checks whether the specified event name has been discarded from Dashboard. If it is,
+            * then create a pending error, and abort.
+            *
+            * @param name The event name
+            * @return Boolean indication whether the event name has been discarded from Dashboard
+        */
+        public UnityNativeValidationResult IsEventDiscarded(string eventName)
+        {
+            if (string.IsNullOrEmpty(eventName))
+            {
+                return new UnityNativeValidationResult();
```

Comment:
If the eventName is null or empty, return an error 510.
Do this on clean event name as well.

Reply:
done
 return new UnityNativeValidationResult(510, $"event name is null or empty.");